FRINXio/frinx-frontend
# Add devices to workflows

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FD-440` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  | Yes



Added a quick way for sending devices to workflow. Also had to refactor a lot of shared code - I moved most of it to `shared` package (also removed unused code, mostly related to Flow - that's why this PR looks bigger than it is).

HOW TO TEST:
- open device inventory - list of devices
- select one (or more) devices
- click "Send to workflow" button
- you should see modal with workflows labeled `INVENTORY`
- you'll see execute workflow modal

This functionality on itself isn't that cool, but we will have more workflows that would take a list of device IDs as an input.
